keyboards/macro1/config.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2021 Laneware Peripherals 
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4C50 // "LP" = Laneware Peripherals
+#define PRODUCT_ID      0x9999
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Laneware_Peripherals
+#define PRODUCT         Laneware_MACRO_1
```

Comment:
```suggestion
#define MANUFACTURER    Laneware Peripherals
#define PRODUCT         MACRO-1
```

Reply:
Change made (can't use "-" in product definition)

Reply:
Yes you can: https://github.com/qmk/qmk_firmware/blob/master/keyboards/6ball/config.h#L28

Reply:
thanks, updated.